templates/github/.github/workflows/ci.yml.j2

```diff
@@ -71,8 +71,10 @@ jobs:
         run: flake8 --config flake8.cfg
       {%- endif %}
 
+      {% if extra_linting -%}
       - name: Run extra lint checks
-        run: [ -x .ci/scripts/extra_linting.sh ] && .ci/scripts/extra_linting.sh
```

Comment:
Would you mind if just quoting properly would have solved the problem?
maybe: `"run: [ -x .ci/scripts/extra_linting.sh ] && .ci/scripts/extra_linting.sh"`

Reply:
Sorry, i did the wrong quotes in the first place.